buddyboss/buddyboss-platform
# Fixed word breaking in URL preview of activity

### All Submissions:


Fixes #463

### How to test the changes in this Pull Request:

1. Go to 'activity page'
2. Add a URL and see the preview. Can test with https://www.outsideonline.com/2407410/what-is-regenerative-agriculture
3. Post the activity
4. See error

### Proof Screenshots or Video
https://prnt.sc/qk9z0t

### Other information:


### Changelog entry

> Fixed Word breaking issue for URL preview in activity.
